README.md

```diff
@@ -194,6 +184,93 @@ For the room list, the following changes are made
 python ./backend/scripts/massage_csv.py /tmp/SecretPartyExport.csv /tmp/RoomsSheetExport.csv --weight placed:30,art:10
 ```
 
+## Database Manipulation
+
+There are three commands which allow for wiping, creating snapshots, and cloning either production or a specific database. These commands are accessed via `roombaht_ctl`.
+
+### Wipe
+
+This command will fully wipe (via drop / create) the database for the specified environment. Migrations will need to be performed after this so it should be followed by a deployment. Note it is super annoying (if actually possible, under certain circumstances) to undo. So be careful.
+
+```sh
+$ ./scripts/roombaht_ctl <user> <env> wipe
+```
+
+### Snapshot
+
+This command will create a new database using the specified environment as a template. The naming format will be `<ROOMBAHT_DB>-MMDDYYYY-HHMM`.
+
+```sh
+./scripts/roombaht_ctl <user> <env> snapshot
+```
+
+### Clone
+
+This command is super helpful for testing in staging. It will create a new database, using either production on a specified database as the template. Note this can _only_ be run on staging.
```

Comment:
Can you clarify "only run on staging"?
What happens if this is run on the `prod` env?
The staging env would ping the production env and grab a clone to the staging host?

Reply:
It's [gated](https://github.com/Index01/RoomBot/blob/5f3d870c714900d17c7509ac3967213e5e8b90fb/scripts/roombaht_ctl-remote.sh#L196-L198) to not clone into prod. Seemed like a not terrible safety catch.